doc/modules/cross_validation.rst

```diff
@@ -468,43 +368,166 @@ validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
 
-Label-Shuffle-Split
```

Comment:
Should there be a heading for stratified splitters and an explanation of their purpose? I know it is auxiliary to this PR.